src/commands/coin/coin.ts

```diff
@@ -1,7 +1,14 @@
-import { ApplyOptions } from '@sapphire/decorators';
-import { Args, container } from '@sapphire/framework';
-import { SubCommandPluginCommand, SubCommandPluginCommandOptions } from '@sapphire/plugin-subcommands';
-import { Message, MessageEmbed } from 'discord.js';
+import { Command, container } from '@sapphire/framework';
```

Comment:
lol can u rearrange code so it isnt 300 lines of green then 100 lines of red 😓 makes pr reviews easier

Reply:
I'm not sure what you mean by this

Reply:
rn i might see
X X'
Z Y'
Y Z'

gimme 
X X'
Y Y'
Z Z'
lmao probs just rearrange functions - so hard to read walls of green/red when the diff is actually 3 lines or smth